query_optimizer/tests/execution_generator/Index.test

```diff
@@ -51,9 +51,70 @@ ERROR: Bloom Filter index is not yet implemented (1 : 48)
 ...INDEX bloomIndex ON test(int_col) USING BLOOMFILTER
                                            ^
 ==
-# SMA Index creation is not supported using CREATE INDEX.
-CREATE INDEX smaIndex ON test USING SMA
+# Specifying no columns will index all the columns.
+CREATE INDEX smaIndex ON test USING SMA;
+SELECT * FROM test;
 --
```

Comment:
Maybe we can just do a `SELECT COUNT(*) FROM test;` since this SQL statement only tests that the previous statement succeeded. This can then limit the verbose SQL output.